include/eve/detail/function/simd/riscv/bit_cast.hpp

```diff
@@ -0,0 +1,314 @@
+//==================================================================================================
```

Comment:
seems like this should've migrated to simd_cast.

Reply:
Not all: if lmul values for wide matches, we can use simple bitcasts, no changes for size of underlying types.

Reply:
then memcpy should work, you don't need the casts. The compiler should do the right thing.
We don't do this for other platfroms. Maybe we should but we don't. So let's not here.